relay-server/src/services/outcome.rs

```diff
@@ -500,6 +501,7 @@ impl DiscardReason {
             DiscardReason::InvalidReplayEventPii => "invalid_replay_pii_scrubber_failed",
             DiscardReason::InvalidReplayRecordingEvent => "invalid_replay_recording",
             DiscardReason::InvalidReplayVideoEvent => "invalid_replay_video",
+            DiscardReason::ReplayExceededSegmentLimit => "replay_too_long",
```

Comment:
nit: I find "exceeded segment limit" more expressive than "too long", but it's up to you.

```suggestion
            DiscardReason::ReplayExceededSegmentLimit => "replay_segment_limit_exceeded",
```